client/src/components/ClickableLogo.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+import LogoSrc from "../assets/images/logo-wild-gamy-v3.svg"

```

Comment:
To import from the backend use this :

const BASE_URL = import.meta.env.VITE_API_URL;
const logo-wildy-gamy-wg = `${BASE_URL}/logo/logo-wildy-gamy-wg.svg`;

Or logoWildyGamyWg if eslint don't like kebab case

Reply:
i quite don't understand this so we'll see that on monday if that's okay